### DISCUSSION TITLE
Complaints

### DISCUSSION THREAD

**Discussion by Samuel-Jnr** [community/community#18511] [GitHub Education]
I am Samuel Reuel Chinonyerem, a student of university of PortHarcourt, Nigeria, my GitHub student developer pack requires i send proof of academic status. My school have not issued me a school ID yet even after applying.

I have used my School Library Access card, and the application was rejected, also my 200l online course registration form, application also rejected, i have used my Jamb admissions letter and application was still denied, and i also use a letter from my school for SIWES training which is dated October 2021 as i am currently on SIWES training somewhere very far from Rivers State where my school is located, and still my application was rejected, which other documents do you want me to use that i can lay hands on?

I can't get a recent transcript as school is currently on strike.

Attached is a screenshot of my current level course registration form
Please assist me.
![Screenshot_20220612-023816](https://user-images.githubusercontent.com/107227701/173232222-7ec3a2fc-3a07-4463-96a8-f84106dbee0d.png)

**Comment by baysideengineer** [comment]
Hello @Samuel-Jnr, I can understand your frustration. But maybe this is something you need to solve directly with GitHub. Best would be to [contact](https://support.github.com) GitHub and explain your situation. I hope that your school will soon end their strike and that you will get your desired student developer pack. Wish you all the best.

**Reply by Samuel-Jnr** [reply]
Thanks for your concern,  I hope to get it resolved soonest
